### DISCUSSION TITLE
Disable the C menu

### DISCUSSION THREAD

**Discussion by Littleislandbrewing** [frenck/spook#354] [Ideas]
Any Chance of getting a spook to disable the "C" menu.

My son is restarting my instance through his phone just to piss me off at random times of the day.......

**Comment by frenck** [comment]
Spook can currently not do any frontend changes.

**Comment by noxhirsch** [comment]
@Littleislandbrewing As a workaround you could add a code line with something wrong like `blockrestart:` to your configuration.yaml. That way you get an config error instead of a restart and you have to temporarily comment out / remove that wrong code line to restart. Not a pretty solution, but maybe it helps you anyway.
